db/migrate/20221116163116_add_collections_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddCollectionsToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :active, :boolean, default: true
+    add_column :users, :full_access, :boolean, default: true
```

Comment:
Should we default to false or do you think it's fine to keep the defaults such that they extend the current behavior?

Reply:
As of now, creating a full access user is our normal use case so optimized for that.

Reply:
@justinlittman OK, sounds good.